### PR TITLE
LPS-100777

### PR DESCRIPTION
**Pull request by topolik**

### CODE REVIEW

**Comment by topolik** [comment]
ci:test

**Comment by pavel-savinov** [comment]
Just started reviewing :)

File: modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/internal/security/permission/resource/FragmentCollectionDLFileEntryModelResourcePermissionRegistrar.java [hunk 1]
```diff
@@ -50,8 +52,21 @@ public void activate(BundleContext bundleContext) {
...
+						}
+
+						if (_portletResourcePermission.contains(
+								permissionChecker, model.getGroupId(),
+								FragmentActionKeys.MANAGE_FRAGMENT_ENTRIES)) {
+
+							return true;
+						}
+
+						return null;
```
Comment (topolik) [inline modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/internal/security/permission/resource/FragmentCollectionDLFileEntryModelResourcePermissionRegistrar.java:68]: Hi @pavel-savinov, 

I'm not sure about this. Maybe we should return false here as MANAGE_FRAGMENT_ENTRIES is the only permission we need to check. Correct?

**Comment by pavel-savinov** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/78480.

**Pull request closed without merging**